src/components/form/Input/input.module.css

```diff
@@ -0,0 +1,36 @@
+.input {
+    border-radius: var(--amp-default-border-radius);
+    border: 1px solid;
+    border-color: inherit;
+    height: 2.5rem;
+    transition: all .20s ease-in;
+
+    color: var(--amp-colors-neutral-800);
+    font-size: 1rem;
+    font-weight: 400;
+    line-height: 1.5rem; /* 150% */
+    padding: 0 1rem;
+    width: 100%;
+    transition: all .1s ease;
+}
+
+.inputError {
+    border: 1px solid var(--amp-colors-error-border);
+}
+
+.input:focus:enabled {
+    border: 2px solid var(--amp-colors-accent-primary);
+    outline: none;
+}
+
+.input:disabled {
+    background-color: var(--amp-colors-neutral-300);
+    border: 1px solid var(--amp-colors-neutral-400);
+}
+
+.error {
+    color: var(--color-red-600, #DC2626);
```

Comment:
Same as the other comment

Reply:
hmm yeah, this style isn't used yet since we don't support advanced form state yet. Will clean up in a followup pr.